dataactvalidator/filestreaming/schemaLoader.py

```diff
@@ -39,6 +40,7 @@ def loadFields(fileTypeName, schemaFileName):
             # add schema to database
             with open(schemaFileName, 'rU') as csvfile:
                 reader = csv.DictReader(csvfile)
+                col_count = 0
```

Comment:
It's counting the number of FileColumn objects being imported, so technically, columns. I can rename this to something like `filecolumn_count` to make it clearer what we're tracking.